ops/model.py

```diff
@@ -985,6 +1001,55 @@ def location(self):
         return self._location
 
 
+class Container:
+    """Represents a named container in a unit.
+
+    Attributes:
+        name: The name of the container from metadata.yaml (eg, 'postgres').
+    """
+
+    def __init__(self, name, pebble_client=None):
+        self.name = name
+
+        if pebble_client is None:
+            socket_path = '/charm/containers/{}/pebble/.pebble.socket'.format(name)
+            pebble_client = pebble.PebbleClient(socket_path=socket_path)
+        self._pebble = pebble_client
+
+    @property
+    def pebble(self) -> 'pebble.PebbleClient':
+        """Return the low-level Pebble client instance for this container."""
+        return self._pebble
+
+    def autostart(self):
+        """Autostart all default services."""
+        self._pebble.autostart_services()
+
+    def start(self, *service_names: typing.List[str]):
+        """Start given service(s) by name."""
+        self._pebble.start_services(service_names)
+
+    def stop(self, *service_names: typing.List[str]):
+        """Stop given service(s) by name."""
+        self._pebble.stop_services(service_names)
+
+    def add_layer(self, setup: typing.Union[str, typing.Dict, 'pebble.Setup']):
+        """Dynamically add a service (Pebble) setup layer.
+
+        Args:
```

Comment:
Is there a convention in place for this already?  I remember we discussed this during one of our calls, and it felt a bit awkward that we say "Attributes", but "Args". If we still have a choice, I'd spell it out as "Arguments:" or "Parameters:' here as well, or go the opposite way and shorten them all.

Reply:
Yeah, `Args:` is used everywhere throughout the project. We're using the "google" style for our docstrings, which uses `Attributes:` and `Args:` for whatever reason: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html